extensions/ql-vscode/src/run-queries-shared.ts

```diff
@@ -79,9 +79,6 @@ export class QueryEvaluationInfo {
     return path.join(this.querySaveDir, 'results.dil');
   }
 
-  /**
-   * Get the path that the compiled query is if it exists. Note that it only exists when using the legacy query server.
```

Comment:
Why did you remove this comment?

Reply:
I can add it back. I had originally thought that the new query server would be working similarly to the old one and create a qlo file, but I understand now that it will create the dil file directly.